site/docs-md/basics/configuring-your-app.md

```diff
@@ -29,9 +29,18 @@ The current ones you might configure are:
 
 ```json5
 {
+  // App domain identifier, typically in reverse-DNS notation.
```

Comment:
I don't think "App domain identifier" means something the users can understand.

In the getting started guide, when we tell users to enter the app id, we say "(the package name for Android and the bundle identifier for iOS)"